### PR TITLE
Edit introduction paragraph for gentler intro

### PR DESCRIPTION
**Pull request by aflaxman**
I love this project you have here, and I wonder if you would be open to edits as I read through it.  For my audience, Docker will perhaps not be such a familiar technology, so I've tried to introduce Singularity without comparing it to Docker first.

<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
checkout@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

**Comment by jcohen02** [comment]
Hi @aflaxman, thanks for the contribution and positive feedback on the project 🙂.

We're very happy to receive further edits. If you are planning to make small updates such as fixing typos or making minor enhancements/corrections to the descriptions to improve readability/understandability of the material, that's great.

If you'd like to make more substantial contributions in terms of changing the structure, adding new material or revising the descriptions, I'd strongly advise opening an issue first, explaining what you plan to do. We can then discuss between the maintainers of the content and offer any feedback or suggestions before you put the time in to make changes.

In terms of your point about people not being familiar with Docker, this is an important point and one that's currently a bit of a challenge with this material. The maintainers of this lesson have generally been teaching the material as part of a 2-day course covering both Docker and Singularity (e.g. [https://www.archer2.ac.uk/training/courses/220119-containers/](https://www.archer2.ac.uk/training/courses/220119-containers/)), this ensures that learners have some familiarity with Docker prior to starting this module. Although we are, of course, are very keen for others to make use of the material in whatever way works best for them.

A more fundamental issue is that we find that the vast majority of attendees at our courses are Windows and Mac users. To provide an environment where they can build Singularity images from scratch in an environment where they have root privileges, we opted to use the Docker Singularity images hosted on [quay.io](https://quay.io/repository/singularity/singularity). The later stages of the lesson are, therefore, based on this approach. We then end up requiring Docker knowledge to be able to cover this material. I see a few options here but I guess I should write this up in a general issue so that others can more easily discover it and add thoughts and comments rather than leaving the discussion as part of this PR.

With reference to your revised text, I'll add a review with suggestions but just a couple of points to highlight - we've worked hard with the [companion docker lesson](https://github.com/carpentries-incubator/docker-introduction) to try and ensure complete consistency in the terminology we use around containers and images. As regular users of these tools we found that we sometimes use these terms interchangably and while we know what we mean, we found that this can cause confusion for learners who are completely new to the concepts. In this regard, I'd suggest a couple of minor updates to your text. Also, I wonder if we can make the text a little more general so that it's not focused specifically on research projects (although these can be used as examples).

Anyway, thanks again for the contribution, I appreciate your interest in the project and do feel free to suggest further updates.

File: _episodes/01-singularity-gettingstarted.md [hunk 1]
```diff
@@ -44,7 +44,9 @@ Working with Singularity containers:
...
+[Singularity](https://sylabs.io/singularity/) is a container platform that allows researchers to easily share their work with others by packaging and deploying their software applications in a portable and reproducible manner. When you download a container created with Singularity, you essentially receive a virtual computer that is preconfigured with all of the necessary software and libraries for a specific research project. This saves you the time and effort of setting up a new computer from scratch, as you can simply run the container and have a virtual environment that is identical to the one used by the original researcher. Container platforms like Singularity provide a convenient and consistent way to access and run research projects, as they allow users to isolate their software environments from the host operating system and run multiple experiments simultaneously.
```
Comment (jcohen02) [inline _episodes/01-singularity-gettingstarted.md]: Suggesting some revisions to the text based on comments in the PR conversation. Feel free to accept as is, or to use these changes as suggestions and update manually.

```suggestion
[Singularity](https://sylabs.io/singularity/) is a container platform that allows software engineers and researchers to easily share their work with others by packaging and deploying their software applications in a portable and reproducible manner. When you download a Singularity container image, you essentially receive a virtual computer disk that contains all of the necessary software, libraries and configuration to run one or more applications or undertake a particular task, e.g. to support a specific research project. This saves you the time and effort of installing and configuring software on your own system or setting up a new computer from scratch, as you can simply run a Singularity container from the image and have a virtual environment that is identical to the one used by the person who created the image. Container platforms like Singularity provide a convenient and consistent way to access and run software and tools. Singularity is increasingly widely used in the research community for supporting research projects as it allows users to isolate their software environments from the host operating system and can simplify tasks such as running multiple experiments simultaneously.
```

File: _episodes/01-singularity-gettingstarted.md [hunk 2]
```diff
@@ -44,7 +44,9 @@ Working with Singularity containers:
...
+
+Some users might be familiar with another container platform, called Docker, and in some ways Singularity is similar to Docker, but in others, particularly in the system's architecture, it is fundamentally different. These differences mean that Singularity is particularly well-suited to running on distributed, High Performance Computing (HPC) infrastructure, as well as a Linux laptop or desktop!
```
Comment (jcohen02) [inline _episodes/01-singularity-gettingstarted.md]:
```suggestion
You may be familiar with Docker, another container platform that is now used widely. If you are, you will see that in some ways, Singularity is similar to Docker. However, in others, particularly in the system's architecture, it is fundamentally different. These differences mean that Singularity is particularly well-suited to running on distributed, High Performance Computing (HPC) infrastructure, as well as a Linux laptop or desktop! 

_Later in this material, when we come to look at building Singularity images ourselves, we will make use of Docker to provide an environment in which we can run Singularity with administrative privileges. In this context, some basic knowledge of Docker is strongly recommended. If you are covering this module independently, or as part of a course that hasn't covered Docker, you can find an introduction to Docker in the "[Reproducible Computational Environments Using Containers: Introduction to Docker](https://carpentries-incubator.github.io/docker-introduction/index.html)" lesson._
```

**Review by jcohen02 — CHANGES_REQUESTED** [review]
Thanks for your contribution @aflaxman. I've suggested some updates to your text based on the comments I added in the PR discussion.

**Comment by aflaxman** [comment]
Super, I've accepted your suggestions, and I'll create an Issue if I have any other substantive changes as I work through your Singularity and Docker lessons.

**Review by jcohen02 — APPROVED** [review]
Many thanks for the contribution @aflaxman, I'll get this merged in now.

**Pull request merged**